Formula/m/manim.rb

```diff
@@ -22,21 +22,34 @@ class Manim < Formula
   depends_on "cython" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+
   depends_on "cairo" # for cairo.h
   depends_on "ffmpeg"
   depends_on "fontconfig"
   depends_on "freetype"
   depends_on "glib"
+  depends_on "jpeg-turbo"
+  depends_on "libtiff"
+  depends_on "little-cms2"
   depends_on "numpy"
+  depends_on "openblas"
   depends_on "pango"
   depends_on "pillow"
   depends_on "py3cairo"
   depends_on "python@3.12"
   depends_on "scipy"
 
+  uses_from_macos "zlib"
+
   on_macos do
+    depends_on "gcc"
     depends_on "gettext"
     depends_on "harfbuzz"
+    depends_on "libimagequant"
+    depends_on "libraqm"
+    depends_on "libxcb"
+    depends_on "openjpeg"
+    depends_on "webp"
```

Comment:
You should be able to get rid of these if you remove the `pillow` resource.

Reply:
cc @branchvincent for thoughts

Reply:
sounds good to me, I think #171786 just missed updating `pypi_formula_mappings.json` (same for `scipy`)